server/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const PORT = 4000;
```

Comment:
We could have the port as an environment variable, something like `process.env.PORT` and a fallback value. 
something like `PORT = process.env.PORT || 4000` makes things easier when deploying this on a server like heroku.